templates/default/web_app.conf.erb

```diff
@@ -3,10 +3,10 @@
 #  Local modifications will be overwritten by Chef.
 #
 <VirtualHost *:<%= node['stash']['apache2']['port'] %>>
-	<% unless node['stash']['apache2']['virtual_host_name'].empty? -%>
```

Comment:
And just confirmed that chef-sugar would resolve this. That cookbook is pretty established and has lots of other things that wemight benefit from using, so perhaps we could add it?

https://github.com/sethvargo/chef-sugar/blob/bd65556f7c101d9fa8cd68d6a7ded2a1daf6525b/lib/chef/sugar/core_extensions/object.rb#L21